README.md

```diff
@@ -1049,6 +1049,66 @@ by passing the html5 option:
 <%= f.input :expires_at, as: :date, html5: true %>
 ```
 
+### Using non Active Record objects
+
+There are few ways to build forms with objects that don't inherit from Active Record, as 
+follow:
+
+You can include the module `ActiveModel::Model`.
+
+```ruby
+class User
+  include ActiveModel::Model
+
+  attr_reader :name, :age
+end
+```
+
+If you are using Presenters or Decorators that inherit from `SimpleDelegator` you can delegate 
+it to the model.
+
+```ruby
+class UserPresenter < SimpleDelegator
+  # Without that, Simple Form will consider the user model as the object.
+  def to_model; self; end
+end
+```
+
+You can define all methods required by the helpers.
+
+```ruby
+class User 
+  extend ActiveModel::Naming
+
+  def id; end;
```

Comment:
We don't need the semicolon after the `end` ✂️. There are other points we have to fix this, take a look at them too. 😉